docs/getting-started/testnet-azure-1click.md

```diff
@@ -201,11 +201,11 @@ If you haven't installed `chain-maind` yet, please follow [Step 1. Get the Crypt
 
 ```bash
 $ chain-maind version
-0.8.0-rc1
+0.8.0-croeseid
 ```
 
 - Testnet binary for
-  [Mac](https://github.com/crypto-com/chain-main/releases/download/v0.8.0-rc1/chain-main_0.8.0-rc1_Darwin_x86_64.tar.gz) and [Windows](https://github.com/crypto-com/chain-main/releases/download/v0.8.0-rc1/chain-main_0.8.0-rc1_Windows_x86_64.zip) are also available.
+  [Mac](https://github.com/crypto-com/chain-main/releases/download/v0.8.0-croeseid/chain-main_0.8.0-croeseid_Darwin_x86_64.tar.gz) and [Windows](https://github.com/crypto-com/chain-main/releases/download/v0.8.0-croeseid/chain-main_0.8.0-croeseid_Windows_x86_64.zip) are also available.
```

Comment:
@allthatjazzleo Would you mind to update the links in here as well ? https://chain.crypto.com/docs/getting-started/croeseid-testnet.html#step-1-get-the-crypto-com-chain-binary

Reply:
sure